yt/frontends/swift/io.py

```diff
@@ -65,8 +65,13 @@ def _get_smoothing_length(self, sub_file, pdtype=None, pshape=None):
         with h5py.File(sub_file.filename, mode="r") as f:
             pcount = f["/Header"].attrs["NumPart_ThisFile"][ind].astype("int64")
             pcount = np.clip(pcount - si, 0, ei - si)
+            keys = f[ptype].keys()
+            # SWIFT commit a94cc81 changed from "SmoothingLength" to "SmoothingLengths"
```

Comment:
does SWIFT have actual version numbers too ? if so, I'd suggest documenting the versions before and after this commit. Otherwise, that's probably the best we can do.